frontend/app/dashboard/src/views/webshops/WebshopsMenu.vue

```diff
@@ -0,0 +1,263 @@
+
+      <!-- other -->
+      <template
+        v-if="
+          enableWebshopModule &&
+          ((fullAccess && hasWebshopArchive) || canCreateWebshops)
+        "
+      >
+        <hr />
+        <STList>
+          <!-- archive -->
+          <STListItem
+            v-if="fullAccess && hasWebshopArchive"
+            element-name="button"
+            :selectable="true"
+            :class="{ selected: isSelected(Button.Archive) }"
+            @click="$navigate(Routes.Archive)"
+          >
+            <template #left><span class="icon archive"></span></template>
+            <h2 class="style-title-list">Archief</h2>
+          </STListItem>
+
+          <!-- add webshop -->
+          <STListItem
+            v-if="canCreateWebshops"
+            element-name="button"
+            :selectable="true"
+            @click="addWebshop()"
+          >
+            <template #left><span class="icon add"></span></template>
+            <h2 class="style-title-list">Webshop</h2>
+          </STListItem>
+        </STList>
+      </template>
+    </main>
+  </div>
+</template>
+
+<script setup lang="ts">
+import {
+  defineRoutes,
+  useNavigate,
+  useUrl,
+} from "@simonbackx/vue-app-navigation";
+import { useContext, useOrganization } from "@stamhoofd/components";
+import {
+  AccessRight,
+  WebshopPreview,
+  WebshopStatus,
+} from "@stamhoofd/structures";
+import { Formatter, Sorter } from "@stamhoofd/utility";
+import { ComponentOptions, computed, onActivated, ref } from "vue";
+
+//#region composables
+const $organization = useOrganization();
+const $context = useContext();
+const $navigate = useNavigate();
+const urlHelpers = useUrl();
+//#endregion
+
+//#region refs
+const selectedWebshop = ref<string | null>(null);
+const currentlySelected = ref<string | null>(null);
+//#endregion
+
+//#region enums
+enum Button {
+  AddWebshop = "add-webshop",
+  Archive = "webshop-archive",
+  Signup = "signup",
+}
+//#endregion
+
+//#region computed
+const enableWebshopModule = computed(
+  () => $organization.value?.meta.modules.useWebshops ?? false
+);
+
+const fullAccess = computed(
+  () => $context.value?.organizationAuth.hasFullAccess() ?? false
+);
+
+const allWebshops = computed(() => {
+  if (!$organization.value) return [];
+  return $organization.value.webshops;
+});
+
+const visibleWebshops = computed(() =>
+  allWebshops.value
+    .filter((webshop) => webshop.meta.status !== WebshopStatus.Archived)
+    .sort((a, b) =>
+      Sorter.stack(
+        Sorter.byBooleanValue(b.isClosed(), a.isClosed()),
+        Sorter.byStringValue(a.meta.name, b.meta.name)
+      )
+    )
+);
+
+const canCreateWebshops = computed(() =>
+  hasAccessRight(AccessRight.OrganizationCreateWebshops)
+);
+
+const hasWebshopArchive = computed(() =>
+  allWebshops.value.some(
+    (webshop) => webshop.meta.status == WebshopStatus.Archived
+  )
+);
+//#endregion
+
+//#region lifecycle
+onActivated(() => {
+  urlHelpers.setTitle("Verkoop");
+});
+//#endregion
```

Comment:
If we set the title property of the <STNavigationBar /> this will happen automatically and we can drop this 😊

```suggestion
```